docs/docs/unreachable.md

```diff
@@ -0,0 +1,198 @@
+replacement occurs ~ 11 mins after receiving unreachable status for the task.  The reconciliation interval has the same
+effect on the expunging of a task.
+
+
+## DC/OS Unreachable Task Scenarios
+
+The follow scenarios are given for a task going unreachable given a default configuration of a DC/OS cluster. This
+assumes that there are resources with required constraints available in the cluster for the replacement task. All marked
+time references are from the actual loss of an agent from the cluster.
+
+### Scenario 1: UnreachableStrategy `{0, 0}`, 1 agent becomes unresponsive for 4 minutes
+
+- 12:00:00 - Agent 1 stops responding due to a network partition (and is running one task).
+- 12:04:00 - The network partition is resolved, and agent 1 starts responding again.
+
+At a task level, Marathon is not made aware that the agent was momentarily not responding; Mesos does not report the
+task as unreachable.
+
+### Scenario 2: UnreachableStrategy `{0, 0}`, 1 agent becomes unresponsive for 6 minutes
+
+- 12:00:00 - Agent 1 stops responding due to a network partition (and is running one task).
+- 12:05:00 - The Mesos master marks Agent 1 as inactive, and publishes a `TASK_UNREACHABLE` status update.
+- 12:05:03 - Shortly after, Marathon replaces 1 tasks for with the `TASK_UNREACHABLE` status was received.
+- 12:06:00 - Agent 1 starts responding again. The Mesos master marks the agent as reachable, and sends a `TASK_RUNNING` status update.
+- 12:06:03 - Marathon kills the previously unreachable task kill.
+
+### Scenario 3: UnreachableStrategy `{0, 300}`, lose 1 agent for 5 minutes; agent becomes reachable after 6 minutes of original event.
+
+- 12:00:00 - Agent 1 becomes unreachable due to a network partition (and is running 1 task).
+- 12:05:00 - The Mesos master marks Agent 1 as inactive, and publishes a `TASK_UNREACHABLE` update for the task associated with it.
+- 12:05:03 - Marathon launches a replacement task for which the `TASK_UNREACHABLE` status was received.
+- 12:06:00 - Agent 1 becomes starts responding again, and the Mesos master marks it as active. The task associated with
+agent 1 is still running, so a `TASK_RUNNING` status update is published for it.  Marathon shows 2 tasks for an
+application with a target instance count of 1.
+- Between 12:10:00 to 12:20:00 - the reachable task is killed, and Marathon reports 1 of 1 for application.
+
+**Note:** The expunge time is 300 seconds which is 5 minutes. 5 minutes after Marathon was notified of the unreachable
+event is the 10 minute mark.  It is possible that the reconciliation internal just happened prior to the 10 minute mark
+and the next time to expunge is the next reconciliation time which is at the 20 minute mark.
+
+### Scenario 4: UnreachableStrategy `{0, 0}`, Lose 4 nodes for hours, the task node is last.
+
+- 12:00:00 - 4 nodes are lost
+- 12:05:00 - Agent 1 is marked inactive
```

Comment:
I'm not sure Mesos has a notion of an agent being inactive. Marathon has inactive tasks but not Mesos.